ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,18 @@
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+get_smaller = lambda a , b: min(a , b)
+print(get_smaller(5 , 10))
+def cube(x): return x**3
+print(cube(5))
+def absolute_difference(a , b): return (a - b)
+print(absolute_difference(5 , 3))
+def squared_difference(a , b): return ((a - b)**2)
+print(squared_difference(3 , 2))
+def hours_to_minutes(hours): return (hours*60)
+print(hours_to_minutes(2))
```

Comment:
Please call these functions with keyword arguments, not positional arguments:

---

![image](https://user-images.githubusercontent.com/16143771/148816004-c39002ad-cb9f-4777-9b71-889a262546da.png)